libs/core-styles/src/lib/_imports/objects/o-site.css

```diff
@@ -10,17 +10,13 @@ Markup: o-site.html
 Styleguide Objects.Site
 */
 
-/* SEE: https://css-tricks.com/couple-takes-sticky-footer/#article-header-id-3 */
+/* IDEA: Use or add grid solution */
+/* https://github.com/DesignSafe-CI/portal/blob/215915c/designsafe/static/styles/main.css#L935 */
+
+/*
 .o-site {
-  display: flex;
-  flex-direction: column;
-}
-.o-site__body {
-  flex-grow: 1;
-  flex-shrink: 0;
-  flex-basis: auto;
-}
-.o-site__head,
-.o-site__foot {
-  flex-shrink: 0;
+  display: grid;
+  height: 100vh;
+  grid-template-rows: auto 1fr auto;
 }
+*/
```

Comment:
Not using `o-site` (and child classes) because I found [a simpler way to make a sticky footer](https://github.com/TACC/tup-ui/pull/18/files#diff-7a0f61a98c72dc6bd12575fbe90c75fa33d020500b696faccf41904e63bfad2fR65-R70).

But this class may still be useful to stretch main content, [as is done for DesignSafe](https://github.com/DesignSafe-CI/portal/blob/215915c/designsafe/static/styles/main.css#L935).